tests/gtest/aviflosslesstest.cc

```diff
@@ -60,6 +56,14 @@ TEST(BasicTest, EncodeDecodeMatrixCoefficients) {
       testutil::AvifRwData encoded;
       avifResult result =
           avifEncoderWrite(encoder.get(), image.get(), &encoded);
+
+#if !defined(AVIF_ENABLE_YCGCO_R_ENCODING)
+      if (matrix_coefficient == AVIF_MATRIX_COEFFICIENTS_YCGCO_RE) {
+        ASSERT_NE(result, AVIF_RESULT_OK);
```

Comment:
Optional: We should be able to assert `result` is `AVIF_RESULT_NOT_IMPLEMENTED`.

Note: It seems that we can use `EXPECT` instead of `ASSERT` here.